Src/StartMenu/StartMenuDLL/MenuContainer.cpp

```diff
@@ -6252,8 +6252,8 @@ LRESULT CMenuContainer::OnDestroy( UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL
 		m_ScrollTheme=NULL;
 	}
 
-    if (s_pFrameworkInputPane && m_InputCookie)
```

Comment:
Looks like unintended change.

Reply:
This one was incorrectly indented (It used spaces while the rest used tabs)

Reply:
Sure, but why to touch it?

Reply:
Well, I came a across it and it wasn't consistent. Do you want me to undo it?

Reply:
Yes, please, remove those unneeded changes.
Commit should contain only relevant changes.

If you want to improve tab-space consistency that's definitely welcome.
But please do it in separate commit and ideally for whole file(s).

Feel free to do it even within this PR.
So basically you will end up with two commits - one that adds those options for menu aligning and one that fixes tab-space indentation.

Reply:
I have now changed the PR to only add the new options